app/assets/stylesheets/battles.scss

```diff
@@ -6,6 +6,55 @@
   display: none;
 }
 
+//our additions changes
+
+body {
+  background-color: #2F3F73;
+}
+
+.color-0 {
+  background-color: #BFCCC9;
+}
+
+.color-1 {
+  background-color: #EBFBF8;
+}
+
 .avatar {
-  width: 50px;
+  width: 70px;
```

Comment:
1.) Looks like we have some weirdness going on with the indentation in this class
2.) If possible, we should avoid styling pixel values.  Let's try to rely on Foundation classes to size our elements (A ```small-1``` sized div might be about the same size you're hoping to see in your layout and will be much more flexible for a responsive design)